### PR TITLE
iOS: Treat the initial Info.plist string as a mustache template and use the project name to initialize the bundle name and identifier

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

**Comment by chinmaygarde** [comment]
Fixes https://github.com/flutter/flutter/issues/1340

**Comment by abarth** [comment]
LGTM

File: packages/flutter_tools/lib/src/ios/setup_xcodeproj.dart [hunk 1]
```diff
@@ -171,11 +155,11 @@ final String _infoPlistInitialContents = '''
 	<key>CFBundleExecutable</key>
 	<string>Runner</string>
 	<key>CFBundleIdentifier</key>
-	<string>io.flutter.runner.Runner</string>
+	<string>com.example.{{projectName}}</string>
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/ios/setup_xcodeproj.dart]: FYI, we'll need to adapt our logger to pull the strings to listen for out of the plist when we run.

Comment (chinmaygarde) [inline packages/flutter_tools/lib/src/ios/setup_xcodeproj.dart]: Good catch. I will just grep by the executable name since that remains constant.

**Pull request merged**